packages/analytics-browser/src/config.ts

```diff
@@ -121,15 +121,11 @@ export class BrowserConfig extends Config implements IBrowserConfig {
     this.diagnosticsClient = diagnosticsClient;
 
     // Backward compatibility for fetchRemoteConfig
-    if (fetchRemoteConfig) {
-      if (!this.remoteConfig) {
-        this.remoteConfig = {
-          fetchRemoteConfig: fetchRemoteConfig,
-        };
-      }
-      if (!this.remoteConfig.fetchRemoteConfig) {
-        this.remoteConfig.fetchRemoteConfig = fetchRemoteConfig;
-      }
+    this.remoteConfig = this.remoteConfig || {};
+    if (remoteConfig?.fetchRemoteConfig === false || fetchRemoteConfig === false) {
+      this.remoteConfig.fetchRemoteConfig = false;
```

Comment:
This doesn't seem right to me. IMO, remoteConfig.fetchRemoteConfig should always takes precedence once it's set. 

An edge example is that fetchRemoteConfig = false while remoteConfig.fetchRemoteConfig = true. The current code change will disable remote config, but it should be on